ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -576,6 +576,16 @@ final class Metrics(val registry: MetricRegistry) {
 
       val httpRequest: Timer = registry.timer(Prefix :+ "http_request")
       val websocketRequest: Counter = registry.counter(Prefix :+ "websocket_request")
+
+      object Db {
+        private val Prefix: MetricName = HttpJsonApi.Prefix :+ "db"
+        val fetchByIdFetch: Timer = registry.timer(Prefix :+ "fetch_by_id_left_fetch")
+        val fetchByIdQuery: Timer = registry.timer(Prefix :+ "fetch_by_id_right_query")
+        val fetchByKeyFetch: Timer = registry.timer(Prefix :+ "fetch_by_key_left_fetch")
+        val fetchByKeyQuery: Timer = registry.timer(Prefix :+ "fetch_by_key_right_query")
+        val searchFetch: Timer = registry.timer(Prefix :+ "search_fetch")
```

Comment:
maybe rename to `syncQuery` and eventual timers can be  derived from that e.g `sync_query_left_fetch`.

Reply:
Oh no, did I keep that left and right? :facepalm: Welp, will remove that left and right